accname/name/comp_text_node.html

```diff
@@ -32,7 +32,7 @@ <h2 class="ex" data-expectedlabel="heading label" data-testname="heading with te
 <!-- Todo: test all remaining cases of https://w3c.github.io/accname/#comp_text_node -->
 
 <script>
-AriaUtils.verifyLabelsBySelector(".ex");
+AriaUtils.verifyLabelsBySelector(1, ".ex");
```

Comment:
See below... Keep `comp_text_node.html` as-is (but with a warning) to make way for https://github.com/web-platform-tests/wpt/pull/42407

Reply:
#42407 was since merged so it's no longer necessary to leave this change out of the PR, but it should get the mentioned argument order change, and will need an update to the `expectedCount` integer.